estuary/models/freshmaker.py

```diff
@@ -20,9 +20,36 @@ class FreshmakerEvent(EstuaryStructuredNode):
     state_reason = StringProperty()
     triggered_by_advisory = RelationshipTo(
         '.errata.Advisory', 'TRIGGERED_BY', cardinality=ZeroOrOne)
-    triggered_container_builds = RelationshipTo('.koji.ContainerKojiBuild', 'TRIGGERED')
+    successful_koji_builds = RelationshipTo('.koji.ContainerKojiBuild', 'TRIGGERED')
+    requested_builds = RelationshipTo('.FreshmakerBuild', 'TRIGGERED')
 
     @property
     def display_name(self):
         """Get intuitive (human readable) display name for the node."""
         return 'Freshmaker event {0}'.format(self.id_)
+
+
+class FreshmakerBuild(EstuaryStructuredNode):
+    """Definition of a Freshmaker build in Neo4j."""
+
+    id_ = UniqueIdProperty(db_property='id')
+    build_id = IntegerProperty(unique=True, required=True)
+    dep_on = StringProperty()
+    event_id = IntegerProperty(required=True)
```

Comment:
Instead of storing the `event_id`, maybe it makes more sense to have a relationship of:
```python
event = RelationshipFrom('.FreshmakerEvent, 'TRIGGERED')
```